### ISSUE TITLE
Edit GitHub Repository

### ISSUE THREAD

**Issue by moza369** [jsxc/jsxc#1125]
in the enumeration of nextcloud program i found a link under developerDocs name that take you to a github repository
and i see the edit part is open for me so i edit the content in https://github.com/jsxc/jsxc/wiki/*
1-go to https://apps.nextcloud.com/api/v1/platform/13.0.4/apps.json
2 -go to 1:developersDocs: "[https://github.com/jsxc/jsxc/wiki/Contributor-Guide"](https://github.com/jsxc/jsxc/wiki/Contributor-Guide%22)
3-edit the content
poc:
go to https://github.com/jsxc/jsxc/wiki/Install-ojsxc-(owncloud
you will see i edit it to test
Impact
the attacker can add some malicious links and make the users download malwares or anything

**Comment by stokito** [comment]
Yes, this is usual behavior for Wiki. You have to always be careful when reading it

**Comment by moza369** [comment]
hi @stokito thanks for your time i appreciate your response.
